<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpikeMap</class>
 <widget class="QDialog" name="SpikeMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Probe Index:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="probe_ind_spinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>N_ch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="n_cs_spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>383</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ch_start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="lead_chan_spinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>global_start (min)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="global_start_spinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>local_dur (sec)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="local_spinBox">
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>bin_width_ms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="bin_width_ms_spinBox">
        <property name="singleStep">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="globalMap_radioButton">
        <property name="text">
         <string>Global</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="recentMap_radioButton">
        <property name="text">
         <string>Local</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="spikeMap_startStop_button">
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCustomPlot" name="spikeMap_qcp" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
